### ISSUE TITLE
Missing license

### ISSUE THREAD

**Issue by hyperknot** [MapQuest/mq-carto-style-vector#20]
Is this going to be an open source style? Or proprietary? If open source, can you possibly add a license?

**Comment by klokan** [comment]
@pnorman could you please clarify the license of this repo?

**Comment by pnorman** [comment]
No, I no longer work for MapQuest so can't issue comments on their behalf.
